.gitignore

```diff
@@ -11,6 +11,4 @@ yarn.lock
 
 .env
 
-dist
```

Comment:
You need to leave `dist` in the `.gitignore`, as this folder is generated by the typescript compiler and should not be commited

Reply:
That's what I thought, but I don't see `dist` anywhere.  When is it generated?

Reply:
At the root of the project when you run the typescript compiler through `tsc`

Reply:
When is `tsc` run?

Reply:
You can run it anytime when you wanna check that your code compiles without error (some errors might not be displayed by the IDE, but will be caught by the TS compiler) or when you wanna transpile your code to JS for deployment / publishing

Reply:
Okay, fixed.